models/DLICV/README.md

```diff
@@ -1 +1,49 @@
-# DLICV Models
\ No newline at end of file
+## DLICV: ICV (Intra-Cranial Volume) Mask Segmentation
+
+## Description
+
+DLICV is a method developed for segmenting a binary mask of the the intra-cranial region from a T1-weighted brain image. It can be applied for ***estimating the intra-cranial volume (ICV)***, a variable commonly used for correcting regional volumes against inter-individual differences in the head size *[1]*. It can also be applied as a pre-processing step, to calculate a ***brain mask*** that is used as input to following processing steps.
+
+## Usage
+
+The DLICV model can be easily applied for segmenting the ICV on a single or multiple T1 images using the tools provided as part of the [DeepMRSeg package](https://github.com/CBICA/DeepMRSeg)
+
+```
+# After installing DeepMRSeg
+
+# Download pre-trained model
+deepmrseg_downloadmodel --task dlicv
+
+# Process single subject
+deepmrseg_apply --task dlicv --inImg subj1_T1.nii.gz --outImg subj1_T1_DLICV.nii.gz
+
+# Batch processing of multiple subjects using a subject list
+#   User provides a csv file with columns: ID,InputT1,OutputImage
+deepmrseg_apply --task dlicv --sList subjectList.csv
+
+# Batch processing of multiple subjects in input folder 
+#   Testing is applied individually to all images with the given suffix in the input folder
+deepmrseg_apply --task dlicv --inDir myindir --outDir myoutdir --inSuff _T1.nii.gz --outSuff _DLICV.nii.gz
+```
+
+
+## Method Overview
+
+Current methods typically use the T1-weighted image for brain extraction, as T1 is the most widely available image modality for volumetric analyses. However, T1-based brain extraction methods are not very successful in correctly segmenting the cortical CSF, resulting in a brain mask that may be "tight around the brain", i.e. one that does not segment properly all of the cortical CSF.
+
+This causes a consistent bias if the brain mask is used for estimating the ICV: It under-estimates it when a subject has high global cortical atrophy, e.g. with aging, and over-estimates it in younger subjects, as the brain mask is more prone to include dura and other non-brain tissues in areas difficult to segment.
+
+T2-weighted images provide a better contrast for delineating the cortical CSF. However, high resolution T2 images may not be always available. Also, multi-modal techniques bring increased complexity, as they require additional preprocessing steps. 
+
+The main motivation for DLICV was based on a simple hypothesis:
+
+*"The actual boundary of the ICV, which is best differentiated on T2 images, can be learned ***using only raw T1 images as input***, if a model is intensively trained using accurate ground-truth segmentations obtained from T2 images."*
+
+Validation experiments have demonstrated that the proposed strategy resulted in accurate ICV masks. The final model was trained on a very rich dataset with n=450 subjects from 15 studies, and their carefully curated/verified reference labels [TODO: ref to paper]
```

Comment:
Do we need to give these details here? These might be better going in the paper? We can keep a shorter version here. Let's discuss tomorrow.

Reply:
I agree, but kept it here for now.